src/icons/Copy.tsx

```diff
@@ -0,0 +1,59 @@
+import classnames, { fill, strokeWidth } from 'classnames/tailwind'
+
+const fillClasses = classnames(
+  fill('fill-primary-dimmed'),
+  strokeWidth('stroke-1.5')
+)
+
+export default function () {
+  return (
+    <svg
+      width="15"
```

Comment:
The copy button isn't square? If it is, let's leave it like this

Reply:
Nope, unfortunately, it's not a square